config/routes.rb

```diff
@@ -18,20 +18,23 @@
 
   namespace :admin do
     resources :users, only: [:index, :show, :update]
+    patch 'user/enable', to: "users#enable"
```

Comment:
Are these two lines (21 and 22) meant to be nested under users? Currently they are not, they are independent of users, which is why `edit` doesn't inherently have an id attached to it, which I think is causing the routes wonkiness above. Nesting them under `:users` might help. Maybe give it a try and check out rails routes and see if your routes allow for an `:id` then e.g. `/admin/users/:id/enable` and `/admin/users/:id/edit` which would be more standard and probably what you want?